python_examples/R0_inference/learn_R.ipynb

```diff
@@ -0,0 +1,2500 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "958a01fd",
+   "metadata": {},
+   "source": [
+    "# Investigating the relationship between the 'basic_reproduction_number' parameter and Rt."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b68399ea",
+   "metadata": {},
+   "source": [
+    "The effective reproduction number characterizes the temporal dynamics of an infectious disease.\n",
+    "\n",
+    "Here I apply the bayesian inference method from branchpro to infer the effective R0 from simulation outputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "3c186640",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import branchpro\n",
+    "import scipy.stats\n",
+    "import statistics\n",
+    "import plotly.graph_objs as go"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "6b8e0ebf",
+   "metadata": {},
+   "source": [
+    "The main parameter required by these models is the serial interval. Since we aim for an overall effective R0 of approximately 3, we searched the literature for information on the serial interval distribution for the original SARS-CoV-2 variant. Estimates of the mean serial interval for this period range from 4.2 to 7.5. On the basis of its use in Menendez (2020), I proceed with an estimated gaussian serial distribution with mean = 4.7 and standard deviation = 2.9 when using fixed serial intervals."
```

Comment:
```suggestion
    "The main parameter required by these models is the serial interval. Since we aim for an overall effective R0 of approximately 3, we searched the literature for information on the serial interval distribution for the original SARS-CoV-2 variant. ....."
```
Looking later in the code, it seems like we are actually using the serial intervals that are read in from the file, rather than the one constructed based on the mean and sd?
Probably easier to just say here that you used the Covid serial intervals which were taken from here (https://github.com/SABS-R3-Epidemiology/transmission-heterogeneity-results/tree/main/data_library/serial_interval)